transport.rb

```diff
@@ -0,0 +1,27 @@
+require_relative 'constants'
+
+class Transport
+  attr_accessor :weight, :speed, :available, :distance
+
+  def initialize(weight, speed, available, distance = nil)
+    @weight = weight
+    @speed = speed
+    @available = available
+    @distance = distance
+  end
+
+  def delivery_time
+    @distance * @speed
+  end
+
+  def is_bike?
+    if @distance <= Constants::BIKE_MAX_DISTANCE &&
```

Comment:
This method looks more like a duck type reference, but here you should simply do something like: `instance_of(Bike)`.

Reply:
Removed.